src/lib/sap_ha/configuration/fencing.rb

```diff
@@ -83,7 +83,9 @@ def list_items(key)
       end
 
       def table_items
-        @devices.each_with_index.map { |e, i| Item(Id(i), (i + 1).to_s, e) }
+        de_items = @devices.each_with_index.map { |e, i| Item(Id(i), (i + 1).to_s, e) }
+        log.debug "table_items #{@devices}"
```

Comment:
are you sure here you want log devices and not `de_items`? from code it looks like later is expected.